diff --git a/lib/build-docker-compose-config.js b/lib/build-docker-compose-config.js
index a1a4f68256b688efecd0da79282bc7e9f1ed1f1c..28cdeeb724f9e681f5eae4c7939a080c658d24fc 100644
--- a/lib/build-docker-compose-config.js
+++ b/lib/build-docker-compose-config.js
@@ -225,6 +225,7 @@ module.exports = function buildDockerComposeConfig( config ) {
 			},
 			'tests-wordpress': {
 				depends_on: [ 'tests-mysql' ],
+				restart: 'on-failure:5',
 				build: {
 					context: '.',
 					dockerfile: 'Tests-WordPress.Dockerfile',
